src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { Block } from 'baseui/block';
+import { HeadingMedium } from 'baseui/typography';
+import React from 'react';
+
+import {
+  Button, Checkbox, FormControl, Input,
+} from '../../../components';
+import { AsyncOperationStatus } from '../../../types';
+
+import useLoginForm from './login-form.hook';
+
+export default function LoginForm(): React.ReactElement {
+  const {
+    formik, status, error,
+  } = useLoginForm();
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      {status !== AsyncOperationStatus.Idle ? (
```

Comment:
This does not sound right - I think if operation is success, then it should show success

Reply:
Done